qmule/src/org/qcmg/qmule/FixCutAdapter.java

```diff
@@ -0,0 +1,101 @@
+package org.qcmg.qmule;
+
+import picard.cmdline.CommandLineProgram;
+import org.broadinstitute.barclay.argparser.Argument;
+
+import picard.cmdline.StandardOptionDefinitions;
+import htsjdk.samtools.util.IOUtil;
+import htsjdk.samtools.util.StringUtil;
+import htsjdk.samtools.SAMException;
+import htsjdk.samtools.fastq.FastqConstants;
+
+import htsjdk.samtools.fastq.FastqRecord;
+import htsjdk.samtools.fastq.FastqWriter;
+import htsjdk.samtools.fastq.FastqWriterFactory;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Date;
+
+public class FixCutAdapter extends CommandLineProgram {
+    static final String USAGE = "A Fastq file may contains empty seqence after cutAdapter,  Here A base N is set to empty sequence with 0 base quality value";
+    
+    @Argument(shortName = StandardOptionDefinitions.INPUT_SHORT_NAME, doc = "Input Fastq file to be fixed. Default is STDIN",optional=true)
```

Comment:
In text visible to the user FASTQ should be either all upper or all lower `FASTQ` or `fastq`